### ISSUE TITLE
PC Rich Presence API on mobile

### ISSUE THREAD

**Issue by bigpuppy9999** [Aliucord/plugin-requests#97]
### Discord Account

Big#5545

### Plugin Description

I want to be able to set a custom Rich Presence through an API or something.

### Go into more detail...

I'm not sure how this would work, but the basic idea is:

Discord <-- Plugin <--  My program

The plugin would setup an endpoint or something to allow for changing rich presence through a program. That's all really.

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by hjroh0315** [comment]
I think the more accurate structure would be

Discord
   ^
   |
Plugin <----- The Plugin's Setting Page
  ^
   |
  v
Webserver
  ^
  |
Third Party Software

**Comment by omxpro** [comment]
> this would be a good idea, but i think it would be easier to just do it so you can manually set the rich presence via settings

I agree I hope this happens one day (btw the rpc should be visible to everyone right?)

**Comment by Mythical-Forest-Collective** [comment]
Didn't Samsung have an intergration with discord or something for RPC a while ago? Maybe it's still in the code or something?

**Comment by omxpro** [comment]
It's not like RPC with buttons

**Comment by Mythical-Forest-Collective** [comment]
Wdym? Ot definitely wouldn't be simple but I'd imagine that'd provide a base for beginning the work on it

**Comment by ItzOnlyAnimal** [comment]
![image](https://user-images.githubusercontent.com/24845294/162600833-9ab3e53c-f3b1-4ef2-9dd1-4c4ea77a2bd4.png)
![image](https://user-images.githubusercontent.com/24845294/162600834-1175c63a-29a8-4030-b941-071010a7b762.png)
You can go digging through samsung source leak to maybe find the code for it but otherwise it's not feasible for a plugin right now

**Comment by Likver** [comment]
What about a port of that RPC google extension? Is that possible to do?

**Comment by Mythical-Forest-Collective** [comment]
> ![image](https://user-images.githubusercontent.com/24845294/162600833-9ab3e53c-f3b1-4ef2-9dd1-4c4ea77a2bd4.png) ![image](https://user-images.githubusercontent.com/24845294/162600834-1175c63a-29a8-4030-b941-071010a7b762.png) You can go digging through samsung source leak to maybe find the code for it but otherwise it's not feasible for a plugin right now

Ah fair enough, that sucks but it is what it is

**Comment by Ushie** [comment]
Due to the existence of alternatives that deliver an experience Aliucord can't achieve by itself, this is no longer desirable
